config/routes.rb

```diff
@@ -6,5 +6,6 @@
     end
   end
 
+  devise_for :users
   root to: 'users#index'
```

Comment:
When I run your app I get an error.

![image](https://user-images.githubusercontent.com/38088818/189455347-e33fa386-f273-42e7-ba5d-97a2d715cc08.png)

This is because your Devise roots `  devise_for :users` should be first on the routes list. Because it is at the end it cause multiple redirects and ultimately crushes the page.


After fixing the routing error, I get another error.

![image](https://user-images.githubusercontent.com/38088818/189455592-7718052e-0504-4303-a1f5-6fa814d231ef.png)

- Kindly look into it and make sure that users can see `sign up` and `log in` links that allow them to register and login successfully.

Reply:
Addressed